log/login.css

```diff
@@ -341,4 +342,112 @@ input {
 
 .social-container a:hover {
   border: 1px solid #4bb6b7;
-}
\ No newline at end of file
+  background-color: #6be1f9;
+}
+
+
+/* Adjustments for screens smaller than 1024px */
+@media (max-width: 1024px) {
+  .container {
+    width: 80%;
+  }
+
+  .overlay-panel {
+    padding: 0 30px;
+  }
+
+  h1 {
+    font-size: 1.8rem;
+  }
+
+  input {
+    padding: 10px;
+    font-size: 1rem;
+  }
+
+  button {
+    padding: 10px 30px;
+    font-size: 1rem;
+  }
+}
+
+/* Adjustments for screens smaller than 768px */
+@media (max-width: 768px) {
```

Comment:
Use <a> and <label>  tags here to reduce their sizes as how remember me and forgot password are aligned in login.html

And also move the login.html page here to log folder 

login.html has multiple <script> tags (duplicate) remove them too